openapi.yaml

```diff
@@ -12992,6 +12992,13 @@ components:
             If you do not live in a country that collects tax, this should be `null`.
           maxLength: 100
           example: ATU99999999
+        euuid:
+          type: string
+          description: >
+            An external unique identifier for this customer.
```

Comment:
Should we use `Account` here instead of `customer`?

Reply:
done